app/controllers/api/v1/participant_declarations_controller.rb

```diff
@@ -71,7 +71,7 @@ def participant_declaration_query_scope
       end
 
       def participant_declaration_for_lead_provider
-        @participant_declaration_for_lead_provider ||= ParticipantDeclaration.for_lead_provider(cpd_lead_provider).find(params[:id])
+        @participant_declaration_for_lead_provider ||= ParticipantDeclaration::ECF.for_lead_provider(cpd_lead_provider).find(params[:id])
```

Comment:
can we add specs for this please?

Reply:
There's already specs for this scenario.. any other scenario you have in mind so I can add specs?

https://github.com/DFE-Digital/early-careers-framework/blob/main/spec/requests/api/v1/participant_declarations_spec.rb#L922
https://github.com/DFE-Digital/early-careers-framework/blob/main/spec/requests/api/v2/participant_declarations_spec.rb#L640
https://github.com/DFE-Digital/early-careers-framework/blob/main/spec/requests/api/v3/participant_declarations_spec.rb#L776

Reply:
Is it to ensure NPQ declarations are no longer returned maybe? 🤔 I'm not sue its worth adding specs for those given we're nuking the data (or have already removed it I'm not sure?)

Reply:
yea fair enough if NPQ is going we can't really test it